test/common/test_common.cpp

```diff
@@ -457,7 +449,7 @@ TEST_F (pointCloudTest, transient_emplace_back)
 
 TEST_F (pointCloudTest, transient_insert_one_element)
 {
-  set_cloud (640*480, PointXYZ (1, 1, 1), 640);
+  cloud.resize (640, 480, PointXYZ (1, 1, 1));
```

Comment:
Since this line is repeated several times, we can pull those tests out into a separate fixture